types/sharedb/index.d.ts

```diff
@@ -313,4 +316,10 @@ interface SubmitRequest {
     channels: string[] | null;
 }
 
+interface GetOpsOptions {
+    opsOptions?: {
+        metadata?: boolean;
```

Comment:
```suggestion
        metadata?: true;
```

Not sure about this but it seems like it can only ever be true. It is false by default so would be odd to explicitly set to false.

Reply:
I think it's weird to force `true` in the type definition. It's perfectly valid to write code that programmatically requests metadata:

```ts
const options = {
  opsOptions: {
    metadata: shouldFetchMetadata(),
  },
};

function shouldFetchMetadata(): boolean {
  // Do complex logic to determine if we want to fetch metadata
}
```